Dockerfile.admin.client

```diff
@@ -1,4 +1,4 @@
-FROM node:16.15.1-alpine as build
+FROM node:18.18.2-alpine3.17 AS build
```

Comment:
To be in sync with the node versions used in the jobs, we might want to update to version 20 herein as well.

Reply:
Thank you! I am using `20.12.1-alpine3.19` now